index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<!-- 
+  This is file is used as the entry point for the Snowpack development server.
+  This file must stay in the root directory so it can be detected by Snowpack.
+ -->
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Council Data Project</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="./src/index-react.js"></script>
+  </body>
+</html>
```

Comment:
For my clarification, the cookiecutter will also need this file + SEO stuff in here right?

https://github.com/CouncilDataProject/cookiecutter-cdp-deployment/blob/main/%7B%7B%20cookiecutter.hosting_github_repo_name%20%7D%7D/web/public/index.html

Reply:
Yes, on the cookie-cutter side of things you will need to create an `index.html` and something similar to the `index-react.tsx` file so the `App` component can be rendered. 

What might differ is where the `index.html` file lives within the project directory depending on what tooling we use for a dev server and to build the app.

For example, I imagine one of the easiest routes would actually be to use the `create-react-app` script in cookie-cutter and then just delete all the boilerplate we don't need and import this version of `App`.

This is also certainly something @tohuynh should chime in on.

Reply:
I guess we could submodule the cookiecutter repo into this repo?

Reply:
I don't think syncing them is a priority for now though. A nice to have for sure but I would add an issue for it to track / keep us aware that we need to sync them eventually.

Reply:
Yeah, I don't think syncing is super necessary either. The main place we need to get those details correct is just in cookie-cutter since that is where the final form of `index.html` and `index-react.tsx` (btw I think it should just be called `index.tsx` in cookie-cutter but there's already an `index.ts` in cdp-frontend).

Reply:
I was also thinking about it more and I'm going to take back what I said about `create-react-app` being easiest. I think that will end being more boilerplate than we need and since we're already using `snowpack` here (which has the huge benefit of being a ridiculously fast dev server and very easy to set up).

The only thing snowpack doesn't do is provide bundling capabilities, so we would have to set up our own webpack or rollup bundling configuration (but I anticipate this wouldn't be too hard to do).

Reply:
Update: I was wrong about this last comment, snowpack can in fact bundle source code out of the box (they use `esbuild`). So I think continuing on with snowpack would be a very solid option.